extensions/android-paging3/src/main/java/com/squareup/sqldelight/android/paging3/OffsetQueryPagingSource.kt

```diff
@@ -19,13 +19,14 @@ import androidx.paging.PagingState
 import com.squareup.sqldelight.Query
 import com.squareup.sqldelight.Transacter
 import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
 internal class OffsetQueryPagingSource<RowType : Any>(
   private val queryProvider: (limit: Long, offset: Long) -> Query<RowType>,
   private val countQuery: Query<Long>,
   private val transacter: Transacter,
-  private val dispatcher: CoroutineDispatcher,
+  private val dispatcher: CoroutineDispatcher = Dispatchers.IO,
```

Comment:
This change needed? I see defaults in the function parameters which create this and in the tests you always pass a value.

Reply:
I just remembered this was something you mentioned in the review of #2203. Figured it couldn't hurt to include it here.

Reply:
> I think we set a default dispatcher of Dispatchers.IO in the core paging extension and then you never see either a coroutine API or an Rx API unless you need to customize the dispatcher (which will be a rare case).

Reply:
Yeah we just need it in the public function that creates this implementation type though. Since this type is internal it always receives a value for the parameter and will never use the default value.

Reply:
oh whoops I thought I did that but I must have missed it.

Reply:
fixed.